packages/table/src/cell/formats/truncatedFormat.tsx

```diff
@@ -132,6 +136,11 @@ export class TruncatedFormat extends React.Component<ITruncatedFormatProps, ITru
         this.setTruncationState();
     }
 
+    public shouldComponentUpdate(nextProps: ITruncatedFormatProps, nextState: ITruncatedFormatState) {
+        return !Utils.shallowCompareKeys(this.props, nextProps)
+            || !Utils.shallowCompareKeys(this.state, nextState);
```

Comment:
why not an `@PureRender` annotation on the class like we usually do?

Reply:
That... probably should work? @cmslewis ?

Reply:
Yep do that!